fix.sh

```diff
@@ -35,3 +35,7 @@ xmlstarlet ed -P \
 	-i '//_:field[@name="phase"]/_:type[@name="TouchpadGesturePhase"]' -t attr -n 'c:type' -v 'gint8' \
 	Gdk-3.0.gir > tmp
 mv tmp Gdk-3.0.gir
+
+xmlstarlet ed -P -L \
+	-i '//*[@c:type="iconv_t"]' -t attr -n 'name' -v 'iconv_t' \
```

Comment:
Currently it add name on any run :(
`type[@c:type="iconv_t" and not(@name)]` seems fix this too